Author: Ole Streicher <debian@liska.ath.cx>
Description: Run compilation in foreground and write log to stdout
 instead of a file.
--- a/install/unix/install3
+++ b/install/unix/install3
@@ -112,8 +112,6 @@
 
 echo ""
 echo "WARNING: MIDAS installation will delete all dependent files."
-echo "         It will run in background and the output will be sent to"
-echo "         the file <$MID_HOME/tmp/install.$$>"
 echo ""
 echo "Do you want to continue [yn]? (y): " $SV_NONL
 unset answ
@@ -138,17 +136,9 @@
 
 # do we have the 'time' command?
 if [ $CMND_YES = 2 ] ; then
-   /bin/bash $MID_INSTALL/install -i > $MID_HOME/tmp/install.$$ 2>&1 &
+   /bin/bash $MID_INSTALL/install -i
 else
-   time /bin/bash $MID_INSTALL/install -i > $MID_HOME/tmp/install.$$ 2>&1 &
+   time /bin/bash $MID_INSTALL/install -i
 fi
 
-echo "Running install in background"
-echo "Results are coming in $MID_HOME/tmp/install.$$"
-if [ "$1" = "-a" ]; then 
-  echo "Waiting for MIDAS installation to finish..."  $SV_NONL
-  wait
-  echo "Finished."
-  echo ""
-fi
 exit 0
--- a/lib/makefile
+++ b/lib/makefile
@@ -18,7 +18,7 @@
 
 include ../local/default.mk
 
-DEV_NULL = >/dev/null 2>&1
+DEV_NULL =
 
 LIBMIDAS = $(LIBDIR)/libmidas.a 
 LIBGMIDAS = $(LIBDIR)/libgmidas.a 
--- a/system/ftoc/makefile
+++ b/system/ftoc/makefile
@@ -12,7 +12,7 @@
 include ../../local/default.mk
 
 DEV_NULL= >/dev/null 2>&1
-# DEV_NULL=
+DEV_NULL=
 
 M = ../exec
 
--- a/system/machine/makefile
+++ b/system/machine/makefile
@@ -16,7 +16,7 @@
 
 include ../../local/default.mk
 
-DEV_NULL= >/dev/null 2>&1
+DEV_NULL=
 
 M = ../exec
 
